.github/ISSUE_TEMPLATE/b-bug-report.md

```diff
@@ -0,0 +1,22 @@
+---
+name: "🐞 Bug Report"
+about: Bug??? Han, bring me a flamethrower 🚒.
+
+---
+
+## 🐞 Bug Description
+<!-- Short and clear how the bug looks like -->
+<!-- It could be a short converstation with the user -->
+
+## Context
```

Comment:
I think the context should be first so people don't forget it (and it puts Expected Behavior right under Bug Description - these two should be together).